percona/controller/pgcluster/controller.go

```diff
@@ -535,6 +539,47 @@ func (r *PGClusterReconciler) reconcileCustomExtensions(cr *v2.PerconaPGCluster)
 		extensionKeys = append(extensionKeys, key)
 	}
 
+	if cr.CompareVersion("2.6.0") >= 0 {
+		var extensionKeysForDeletion []string
+		var extensionsKeysAnnotations []string
+
+		if cr.Spec.Metadata.Annotations != nil {
+			if val, ok := cr.Spec.Metadata.Annotations["custom_extensions_list"]; ok && val != "" {
```

Comment:
You can name the annotation like `custom_extensions`, without the `list` part, it is assumed it is a list since the name is in plural.

Also, don't forget to put `percona.com` prefix for the annotation.